packages/synapse-react-client/src/components/Markdown/widget/MarkdownButton.tsx

```diff
@@ -22,12 +23,25 @@ export default function MarkdownButton(
   }
   const buttonIsCenterAligned = alignLowerCase === 'center'
   const buttonColor = highlight ? 'secondary' : 'neutral'
+  const isExternalLink =
+    widgetParamsMapped.url?.startsWith('http://') ||
+    widgetParamsMapped.url?.startsWith('https://')
```

Comment:
We use this component in a lot of places where React Router is not used (i.e. in SWC)--this will break if we happen to render this in those cases.

Can you also check the value of [`useInRouterContext`](https://api.reactrouter.com/v7/functions/react_router.useInRouterContext.html) and only use the RouterLink if that is also true?